src/components/Sphere/index.tsx

```diff
@@ -45,7 +45,7 @@ const StyledTextSphere = styled('div', {
   shouldForwardProp: prop => !['color', 'fontSize'].includes(prop.toString()),
 })<{ color: string; fontSize?: number }>`
   color: ${({ theme, color }) =>
-    theme.colorsDeprecated[color as Color] ?? color};
+    theme.colors[color as Color]?.textStrong ?? color};
```

Comment:
```suggestion
    theme.colors[color as Color]?.textWeak ?? color};
```

as the text is in middle of sphere in a white background we should make it weak or text will never be readable